cmd/config.go

```diff
@@ -46,13 +56,42 @@ func (config *ConfigDir) Get() string {
 	return config.dir
 }
 
+func (config *ConfigDir) getCertsDir() string {
+	return filepath.Join(config.Get(), certsDir)
+}
+
+// GetCADir - returns certificate CA directory.
+func (config *ConfigDir) GetCADir() string {
+	return filepath.Join(config.getCertsDir(), certsCADir)
+}
+
+// Create - creates configuration directory tree.
+func (config *ConfigDir) Create() error {
+	return mkdirAll(config.GetCADir(), 0700)
+}
+
+// GetMinioConfigFile - returns absolute path of config.json file.
+func (config *ConfigDir) GetMinioConfigFile() string {
```

Comment:
`GetMinioConfigPath()` looks to me like path to minio configuration where some configuration files are stored whereas the function returning a absolute path of `config.json` file.